plugins/modules/aci_l4l7_device_selection_interface_context.py

```diff
@@ -404,6 +450,13 @@ def main():
                             aci.base_url, tenant, contract, graph, node, context
                         ),
                     )
+                elif child.get("vnsRsLIfCtxToInstP") and child.get("vnsRsLIfCtxToInstP").get("attributes").get("tDn") != l3out_tdn:
+                    aci.api_call(
+                        "DELETE",
+                        "{0}/api/mo/uni/tn-{1}/ldevCtx-c-{2}-g-{3}-n-{4}/lIfCtx-c-{5}/rsLIfCtxToInstP.json".format(
+                            aci.base_url, tenant, contract, graph, node, context
+                        ),
+                    )
```

Comment:
Is it not modifiable when created?

Could it be handled with a second object with status=deleted in the same payload?